javaagent-tooling/src/test/groovy/io/opentelemetry/javaagent/tooling/OpenTelemetryInstallerTest.groovy

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.tooling
+
+import io.opentelemetry.api.GlobalOpenTelemetry
+import io.opentelemetry.instrumentation.api.config.Config
+import io.opentelemetry.extension.noopapi.NoopOpenTelemetry;
+import spock.lang.Specification
+
+class OpenTelemetryInstallerTest extends Specification {
+
+
+  void setup(){
+    GlobalOpenTelemetry.resetForTest()
+  }
+
+  void cleanup(){
+    GlobalOpenTelemetry.resetForTest()
+  }
+
+
+  def "should initialize noop"(){
+
+    given:
+    def config = Mock(Config)
+    config.getBooleanProperty(OpenTelemetryInstaller.JAVAAGENT_NOOP_CONFIG, false) >> true
+    config.getBooleanProperty(OpenTelemetryInstaller.JAVAAGENT_ENABLED_CONFIG, true) >> true
+    config.getAllProperties() >> ["otel.javaagent.enabled":"true", "otel.javaagent.experimental.sdk.noop":"true"]
```

Comment:
Nit: `Config` is a fairly simple type, you can create a new instance of it instead of mocking it:
```suggestion
    def config = Config.newBuilder()
        .readProperties([
            (OpenTelemetryInstaller.JAVAAGENT_NOOP_CONFIG): "false",
            (OpenTelemetryInstaller.JAVAAGENT_ENABLED_CONFIG): "true"
        ])
        .build()
```